bindings/python/src/normalizers.rs

```diff
@@ -410,25 +415,55 @@ impl PySequence {
         PyTuple::new(py, [PyList::empty(py)])
     }
 
-    fn __len__(&self) -> usize {
-        0
+    fn __len__(self_: PyRef<'_, Self>) -> usize {
+        match &self_.as_ref().normalizer {
+            PyNormalizerTypeWrapper::Sequence(inner) => inner.len(),
+            PyNormalizerTypeWrapper::Single(_) => 1,
+        }
```

Comment:
nice, thanks! 
I think the issue you mentioned about length comes from here

Reply:
Because it's a `Single`? Not sure, it should deser to empty Sequence, but the type is `Sequence` with length 1 containing an `NKFC`